src/commands/service/get-display-name.ts

```diff
@@ -0,0 +1,75 @@
+import commander from "commander";
+import { read as readBedrockYaml } from "../../lib/bedrockYaml";
+import { build as buildCmd, exit as exitCmd } from "../../lib/commandBuilder";
+import { logger } from "../../logger";
+import decorator from "./get-display-name.decorator.json";
+import { build as buildError, log as logError } from "../../lib/errorBuilder";
+import { errorStatusCode } from "../../lib/errorStatusCode";
+
+export interface CommandOptions {
+  path: string | undefined;
+}
+
+/**
+ * Executes the command, can all exit function with 0 or 1
+ * when command completed successfully or failed respectively.
+ *
+ * @param opts validated option values
+ * @param exitFn exit function
+ */
+export const execute = async (
+  opts: CommandOptions,
+  exitFn: (status: number) => Promise<void>
+): Promise<void> => {
+  // The assumption is that this command should be ran from the directory where bedrock.yaml exists
+  try {
+    if (!opts.path) {
+      throw buildError(
+        errorStatusCode.VALIDATION_ERR,
+        "service-get-display-name-path-missing-param-err"
+      );
+    }
+    const bedrockFile = readBedrockYaml(process.cwd());
+    if (!bedrockFile) {
+      throw buildError(
+        errorStatusCode.FILE_IO_ERR,
+        "service-get-display-name-bedrock-yaml-missing-err"
+      );
+    }
+
+    const serviceIndex = Object.keys(bedrockFile.services).find(
+      (index) => opts.path === bedrockFile.services[+index].path
+    );
+
+    if (serviceIndex) {
+      console.log(bedrockFile.services[+serviceIndex].displayName);
```

Comment:
Just had a though @samiyaakhtar , so `displayName` can be a an empty string. What do we want to return in this case? Is an empty string a valid case?

Reply:
@mtarng I think so, it should be valid but will just show an empty service name in the dashboard. I think that's ok since we want this to be the only source of truth for service name

Reply:
Sounds good.

Reply:
Agree that a validation would be good. But should be noted that `spk service create` now requires a non-empty displayName to be provided

Reply:
@evanlouie in that case we should be good going forward